<log type="Session" name="TEST-DUYENDT-PC:LQA" at="2023-06-15 15:36:53.839"/>
<log type="Test" name="PoM" status="Info"  at="2023-06-15 15:36:54.946"><data type="text">Test Started: C:\Users\LQA\Documents\My Rapise Tests\PoM\Main.js</data>
</log>
<log type="Message" name="Starting scenario: Test" status="Info"  at="2023-06-15 15:36:54.950"/>
<log type="Assert" name="Failure in Test" status="Fail"  at="2023-06-15 15:36:54.954"><data type="text" caption="Exception">Error(-2147024894): </data>
<data type="link" url="C:\Users\LQA\Documents\My Rapise Tests\PoM\Main.js(9,1)" text="C:\Users\LQA\Documents\My Rapise Tests\PoM\Main.js(9,1)"/>
<data type="text">Global.DoRunTestSet(Global.GetFullPath(&quot;Scripts//TestSetCritial.js&quot;));</data>
</log>
<log type="Test" name="PoM" status="Fail" closed="PoM" _guid="undefined"  at="2023-06-15 15:36:54.957" comment="Passed:0 Failed:1 Time:0.121s" />
